monai/networks/nets/cell_sam_wrapper.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import torch
+from torch import nn
+from torch.nn import functional as F
+
+try:
+    from segment_anything.build_sam import build_sam_vit_b
+except ImportError:
+    raise AssertionError("SAM is not installed, please run: pip install git+https://github.com/facebookresearch/segment-anything.git")
+
+
+_all__ = ["CellSamWrapper"]
+
+class CellSamWrapper(torch.nn.Module):
+    """
+    CellSamWrapper is thin wrapper around SAM model https://github.com/facebookresearch/segment-anything
+    with an image only decoder, that can be used for segmentation tasks.
+
+
+    Args:
+        auto_resize_inputs: whether to resize inputs before passing to the network.
+        network_resize_roi: expected input size for the network.
+        checkpoint: checkpoint file to load the SAM weights from.
+        return_features: whether to return features
+
+    """
+
+    def __init__(
+        self,
+        auto_resize_inputs=True,
+        network_resize_roi=[1024, 1024],
+        checkpoint="sam_vit_b_01ec64.pth",
+        return_features=False,
+        *args,
+        **kwargs,
+    ) -> None:
+        super().__init__(*args, **kwargs)
+
+        print(
```

Comment:
Please don't have print statements in code, you can write to the log if you wish though this is typically not necessary.

Reply:
removed. thank you!